From 1933d30528c58598d7423d8b307d8e0fd2c50ad5 Mon Sep 17 00:00:00 2001
From: Tao Bao <tbao@google.com>
Date: Wed, 15 May 2019 10:00:47 -0700
Subject: [PATCH] Revert^2 "Build bootctrl.msm8998 with Android.bp.""

This reverts commit 28b533bd96d92d7a3bf880575b183cfffe05cd12.

Bug: 34254109
Test: Sideload on taimen.
Change-Id: I6a85b7885903df818deb32c40c751ac4358a6dbc
Merged-In: I6a85b7885903df818deb32c40c751ac4358a6dbc
(cherry picked from commit 7ba68ec1d2175905bbfecab5d4a57503bdcb9f76)
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 6029054..4b94b7a 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,5 +1,5 @@
 # Preset TARGET_USES_HARDWARE_QCOM_BOOTCTRL for existing platforms.
-ifneq ($(filter msm8996 msm8998,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8996,$(TARGET_BOARD_PLATFORM)),)
 TARGET_USES_HARDWARE_QCOM_BOOTCTRL := true
 endif
 
-- 
2.15.1

